src/hooks/useReportScrollManager/index.native.ts

```diff
@@ -29,7 +29,21 @@ function useReportScrollManager(): ReportScrollManagerData {
         flatListRef.current?.scrollToOffset({animated: false, offset: 0});
     }, [flatListRef, setScrollPosition]);
 
-    return {ref: flatListRef, scrollToIndex, scrollToBottom};
+    /**
+     * Scroll to the offset of the flatlist.
+     */
+    const scrollToOffsetWithoutAnimation = useCallback(
```

Comment:
Few things:

- I'm a bit confused why this is necessary. You say below:

> This is a workaround because 'autoscrollToTopThreshold' does not always function correctly

But ... is this something we can fix? I'm pretty confused by its usage in `useHandleList` so maybe that will become more clear as I spend more time with this code...

- I see the only place where this is called is in `ReportActionsView` with an offset of `1`. Could we instead just use the existing `scrollToBottom` function from this hook and remove this new function? `scrollToBottom` already using `animated: false` on web and native platforms.

Reply:
> But ... is this something we can fix? I'm pretty confused by its usage in useHandleList so maybe that will become more clear as I spend more time with this code...

This issue is currently under investigation. We can't use the 'scroll to bottom' approach because we need to trigger scroll computation. That's why I'm implementing a scroll of 1 pixel.
Also, I'd like to mention that this isn't noticeable during render time